book/02-git-basics/sections/recording-changes.asc

```diff
@@ -246,7 +246,7 @@ build/
 # ignore doc/notes.txt, but not doc/server/arch.txt
 doc/*.txt
 
-# ignore all .pdf files in the doc/ directory
+# ignore all .pdf files in the doc/ directory or in the sub directories of doc/
```

Comment:
I think it is nicer to have the word "sub directory" in there somewhere. To make the association with '**' stronger and remove any ambiguity. After all that is what it is all about, which you might not know if you were seeing it for the first time.